### ISSUE TITLE
WhiteClover cultivar changes to phenology are suspicious

### ISSUE THREAD

**Issue by hol430** [APSIMInitiative/ApsimX#6524]
The Kopu cultivar of whiteclover makes the following changes:

```
[Phenology].ThermalTime.Response.X = 2.5,25,37.5
[Phenology].ThermalTime.Response.X = 0,22.5,0
```

The first change here is essentially redundant, as it's immediately overwritten by the second change. @HamishBrownPFR / @rcichota is this intentional? Or is the second line supposed to change the y-values?

This also occurs in the tahora and apex cultivars. This seems to have been the case since whiteclover resource was initially added back in 2019 (in 0398f92fbf8e941d1641cec293fd909f05500be4).

**Comment by rcichota** [comment]
Yep, that seem wrong. the two lines is probably because I was me doing and learning things at the same time. I might have had the idea of eventually changing both X and Y and never got to it. Whereas the second line ought to be an Y. I will have a look at it tomorrow and commit changes as needed. Thanks for noticing it.

**Comment by par456** [comment]
This was never fixed, did you want to put into a fix for this or have us handle it? @rcichota

**Comment by rcichota** [comment]
Hi @par456 
Thanks for remind me of this. I will check and fix this soon...
